tests/api/plugins/content-releases/releases.test.api.ts

```diff
@@ -0,0 +1,646 @@
+  attributes: {
+    name: {
+      type: 'string',
+    },
+    description: {
+      type: 'string',
+      required: true,
+    },
+  },
+};
+
+describeOnCondition(edition === 'EE')('Content Releases API', () => {
+  const builder = createTestBuilder();
+  let strapi;
+  let rq;
+  const data = {
+    releases: [],
+  };
+
+  const deleteAllReleases = async () => {
+    const releases = await rq({
+      method: 'GET',
+      url: '/content-releases',
+    });
+
+    await Promise.all(
+      releases.body.data.map(async (release) => {
+        await rq({
+          method: 'DELETE',
+          url: `/content-releases/${release.id}`,
+        });
+      })
+    );
+  };
+
+  const createEntry = async (uid, data) => {
+    const { body } = await rq({
+      method: 'POST',
+      url: `/content-manager/collection-types/${uid}`,
+      body: data,
+    });
+
+    return body;
+  };
+
+  beforeAll(async () => {
+    await builder.addContentTypes([productModel]).build();
+    strapi = await createStrapiInstance();
+    rq = await createAuthRequest({ strapi });
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date('2024-01-01T00:00:00.000Z'));
+
+    // Create products
+    await Promise.all([
+      createEntry(productUID, { name: 'Product 1', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 2' }),
+      createEntry(productUID, { name: 'Product 3', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 4', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 5', description: 'Description' }),
+    ]);
+
+    await deleteAllReleases();
+  });
+
+  afterAll(async () => {
+    jest.useRealTimers();
+    await strapi.destroy();
+    await builder.cleanup();
+  });
+
+  describe('Create Release', () => {
+    test('Create a release', async () => {
+      const res = await rq({
+        method: 'POST',
+        url: '/content-releases/',
+        body: {
+          name: 'Test Release',
+          scheduledAt: null,
+        },
+      });
+
+      data.releases.push(res.body.data);
+
+      expect(res.statusCode).toBe(200);
+    });
+
+    test('cannot create a release with the same name', async () => {
+      const res = await rq({
+        method: 'POST',
+        url: '/content-releases/',
+        body: {
+          name: 'Test Release',
+          scheduledAt: null,
+        },
+      });
+
+      expect(res.statusCode).toBe(400);
+      expect(res.body.error.message).toBe('Release with name Test Release already exists');
+    });
+
+    test('cannot create a scheduled release without timezone', async () => {
+      const res = await rq({
+        method: 'POST',
+        url: '/content-releases/',
+        body: {
+          name: 'Test Scheduled Release',
+          scheduledAt: '2024-10-10T00:00:00.000Z',
+          isScheduled: true,
+          time: '00:00',
+          date: '2024-10-10',
+          timezone: null,
+        },
+      });
```

Comment:
Something that came to my mind now, we still have not standarized how to build api tests.. So this is just me throwing ideas :D 

In tests where the same payload is repeated multiple times, I would build a generic function with default parameters:

```suggestion
	const createRelease = (params: Partial<APIContract>) => {
	  return rq({
        method: 'POST',
        url: '/content-releases/',
        body: {
          name: 'Test Scheduled Release',
          scheduledAt: '2024-10-10T00:00:00.000Z',
          isScheduled: true,
          time: '00:00',
          date: '2024-10-10',
           ...params
        },
      });
	}
   // ....
      const res = await createRelease({ timezone: null })
```

do you think this could be useful here?

Reply:
That way we could have slim and more readable tests
```
it('Create release without timezone', () => {
  const res = await createRelease({timezone: null})
  expect(res.statusCode).toBe(400)
  ...
})

it('Create a scheduled release', () => {
  const res = await createRelease({scheduledAt: date, isScheduled: true, timezone...})
  expect(res.statusCode).toBe(400)
  ...
})